core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1401,24 +1401,20 @@ object GroupMetadataManager {
       val version = buffer.getShort
       val valueSchema = schemaForGroupValue(version)
       val value = valueSchema.read(buffer)
+      val maxVersion = GROUP_VALUE_SCHEMAS.size - 1
```

Comment:
We should add a constant in `GroupMetadataManager` for this, maybe call it `CURRENT_GROUP_METADATA_VALUE_SCHEMA_VERSION` for consistency with the other two similar fields.

Reply:
Good suggestion! Updated.